quisp/modules/QRSA/HardwareMonitor/HardwareMonitor.cc

```diff
@@ -611,8 +612,8 @@ void HardwareMonitor::sendLinkTomographyRuleSet(int my_address, int partner_addr
   std::vector<int> partners = {partner_address};
 
   if (num_purification > 0) {
-    if (purification_type == 2002) {  // Performs both X and Z purification for each n.
-      /// # Purification_type 2002: #
+    if (purification_type == "R1_Single_X-R1_Single_Z") {  // Performs both X and Z purification for each n.
```

Comment:
`R1_SINGLE_X_R1_SINGLE_Z`

* keep the naming convention: CAPITAL_CASE
* no dash `-`